metron-stellar/stellar-common/README.md

```diff
@@ -100,6 +102,28 @@ In the core language functions, we support basic functional programming primitiv
 * `FILTER` - Filters a list by a predicate in the form of a lambda expression.  For instance `FILTER([ 'foo', 'bar'], (x ) -> x == 'foo' )` returns `[ 'foo' ]`
 * `REDUCE` - Applies a function over a list of input.  For instance `REDUCE([ 1, 2, 3], (sum, x) -> sum + x, 0 )` returns `6`
 
+### Stellar Language Match Expression
+
+Stellar provides the capability to write match expressions, which are similar to switch statements commonly found in c like languages, but more like
+Scala's match.
```

Comment:
I'd be a bit hesitant to say this is similar to Scala's match. Our match statement doesn't really support pattern matching.

Reply:
Do you have any suggestions for a better way to describe it?  I'm not super happy with this either, I'll take suggestions ;)

Reply:
@ottobackwards sure. Right now, I think our match statement syntax is just a nicer syntax for if then else blocks. I would just state something along those lines. I know in the future it will be more.